packages/nextjs-mf/src/loaders/nextPageMapLoader.ts

```diff
@@ -33,8 +39,11 @@ export default function nextPageMapLoader(
 
```

Comment:
The function 'getNextPages' can be improved by using a map function instead of a for loop. This can improve readability and maintainability.